### PR TITLE
Organizing UserRepository to distribute all fetched data

### PR DESCRIPTION
**Pull request by JWoltjen**
#### What does  this PR do?

1. This PR organizes the data so that the UserRepository stores an array of users, which are each instantiated user objects. 

2. The linkData method instantiates the users stored in the UserRepository array (this.users) by iterating over the rawData we will eventually retrieve from the server, and passing the data retrieved into a helper function for each data category. 

3. These helper functions, linkHydration, linkSleep, and linkActivity, which are all called within linkData, each iterate over the users array, and forEach user, populates that given user's records array within each repository, HydrationRepository, SleepRepository, ActivityRepository all have this.record as their only property. These helper functions populate those arrays for each user.

#### Where should the reviewer start?

The reviewer should start in UserRepository.js to review the changes mentioned above. 

#### How should this be manually tested?

No tests have been appropriate for this PR. 

#### Any background context you want to provide?

The PR contemplates that all of the data eventually retrieved from the server will be passed into scripts.js and then the DOM through an instantiation of the UserRepository class. Eventually in scripts.js, we will declare currentUser and todaysDate, and then in the asynchronous function, we will assign currentUser to a random index of the userRepository instantiated within getData. 

#### What are the relevant tickets?
N/A
#### Screenshots (if appropriate)
N/A

### CODE REVIEW

File: src/UserRepository.js [hunk 1]
```diff
@@ -1,11 +1,41 @@
...
+import User from './User'
 
 class UserRepository {
-  constructor(date, userData, activityData, hydrationData, sleepData) {
-    this.users = [];
+  constructor(rawData, todaysDate) {
+    this.users = this.linkData(rawData, todaysDate);
+  }
+
+  linkdata(rawData, todaysDate){
```
Comment (peytonjo) [inline src/UserRepository.js:9]: This function will be great once we get it to properly take in the correct Data from our other files

File: src/UserRepository.js [hunk 2]
```diff
@@ -1,11 +1,41 @@
...
-import hydrationData from './data/hydration';
-import activityData from './data/activity';
-import userData from './data/users';
+
+import User from './User'
 
 class UserRepository {
-  constructor(date, userData, activityData, hydrationData, sleepData) {
-    this.users = [];
+  constructor(rawData, todaysDate) {
```
Comment (demaceo) [inline src/UserRepository.js:5]: good idea of consolidating user-, activity-, hydration-, & sleepData into one single parameter (`rawData`)

File: src/scripts.js [hunk 3]
```diff
@@ -2,16 +2,12 @@ import './css/base.scss';
 import './css/styles.scss';
 import './index.js';
 
-import userData from './data/users';
-import activityData from './data/activity';
-import sleepData from './data/sleep';
-import hydrationData from './data/hydration';
+
+import fetchData from //fetch.js file to be created later//
```
Comment (peytonjo) [inline src/scripts.js:6]: Looks like some of the import files were taken out due to where we are using them! This should be fine after some work is put into the appropriate files

**Review by peytonjo — APPROVED** [review]
Great job, I left some comments on each file, check em out!

**Pull request merged**